Cannot find module '.'

I'm fairly sure this is a now dev problem because I ran into this error converting a previously working endpoint to a now serverless function.

The build is failing on `export * from '.'`, inside a library import.

I've created a MVCE here: https://github.com/HorizonShadow/now-error-mcve

Running `now dev`, and navigating to `localhost:3000/api/test` will produce

```
> Built @now/node@latest:api/test.ts [8s]
Cannot find module '.'
Did you forget to add it to "dependencies" in `package.json`?
```

--- a/packages/now-next/package.json
+++ b/packages/now-next/package.json
@@ -25,7 +25,7 @@
     "@types/resolve-from": "5.0.1",
     "@types/semver": "6.0.0",
     "@types/yazl": "2.4.1",
-    "@zeit/node-file-trace": "0.3.0",
+    "@zeit/node-file-trace": "0.3.1",
     "async-sema": "3.0.1",
     "buffer-crc32": "0.2.13",
     "execa": "2.0.4",

--- a/packages/now-node/package.json
+++ b/packages/now-node/package.json
@@ -29,7 +29,7 @@
     "@types/etag": "1.8.0",
     "@types/test-listen": "1.1.0",
     "@zeit/ncc": "0.20.4",
-    "@zeit/node-file-trace": "0.3.0",
+    "@zeit/node-file-trace": "0.3.1",
     "content-type": "1.0.4",
     "cookie": "0.4.0",
     "etag": "1.8.1",

